apps/super-app/src/services/DetailServices.ts

```diff
@@ -56,48 +54,133 @@ export interface FacebookLink {
   text: string;
 }
 
-export function getServiceCardAbouts(): ServiceCardAbout[] {
+export function getServiceCardDetails(): ServiceCardDetail[] {
   return [
     {
       logo: '/images/twc2-logo.png',
       name: 'Transient Workers Count Too',
       shortBlurb:
         'Whatsapp call on Mondays to Fridays (9am - 9pm) or message anytime',
-      route: Path.SupportDetail,
+      route: Path.TWC2,
     },
     {
-      logo: 'https://scontent.fsin16-1.fna.fbcdn.net/v/t39.30808-6/312822980_407024118308030_3649276446816297142_n.jpg?_nc_cat=111&ccb=1-7&_nc_sid=09cbfe&_nc_ohc=rusQliWmleEAX-XViuM&_nc_ht=scontent.fsin16-1.fna&oh=00_AfAzvXYCdtBtRovhSADhgAmeQ31Ev5ZUHiagle1qjfYcoA&oe=63ED310D',
+      logo: 'https://www.probono.sg/wp-content/uploads/2022/12/Pro-Bono-SG-Logo.png',
       name: 'Pro Bono SG',
       shortBlurb:
         'Book in-person appointment with a lawyer and get free basic legal advice',
       route: Path.ProBonoSG,
     },
+    {
+      logo: '/images/irr-logo.png',
+      name: 'ItsRainingRaincoats',
+      shortBlurb: 'Get free eye check up and learn English for free',
+      route: Path.IRR,
+    },
   ];
 }
 
-export function getTwc2Detail(): Twc2SupportDetail {
-  return {
-    logo: '/images/twc2-logo.png',
-    name: 'Transient Workers Count Too (TWC2)',
-    website: 'https://twc2.org.sg/language-menu-for-qr-code/',
-    blurbIntro: `TWC2 helps all foreigners working in Singapore, including those on Work Permits and S-Passes. They help for free. Most common issues are salary non-payment, contract issues and injuries.`,
+export function getSupportDetail(
+  support: string | undefined
+): SupportDetailWithLinks {
+  const supportDetailWithLinks = {
+    logo: '',
+    name: '',
+    website: '',
+    blurbIntro: '',
     blurbFacebookLinks: {
-      title: `Join TWC2's Facebook Community Pages!`,
-      imageSrc: '/images/facebook-icon.svg',
+      title: '',
+      imageSrc: '',
       links: [
         {
-          href: 'https://www.facebook.com/twc2bangla',
-          text: 'Bengali',
+          href: '',
+          text: '',
         },
         {
-          href: 'https://www.facebook.com/twc2tamil',
-          text: 'Tamil',
+          href: '',
+          text: '',
         },
       ],
     },
-    ctaBlurb: 'Call on Mondays to Fridays (9am - 9pm), message any time',
-    ctaButtonText: '+65 6297 7564',
-    ctaLink: 'https://wa.me/6562977564',
-    whatsappLogoSrc: '/images/whatsapp-icon.svg',
+    ctaButtonText: '',
+    ctaLink: '',
+    ctaLogo: '',
   };
+  switch (support) {
+    case 'twc2':
+      supportDetailWithLinks.logo = '/images/twc2-logo.png';
```

Comment:
I strongly feel that this is change brings us in the wrong direction:

a) We want to replace these hardcoded values with Strapi eventually, but this function doesn't emulate a Strapi calling pattern at all. This takes a single string as a parameter and returns 1 of 3 values we care about, but we plan to use Strapi single types for each page, where a single type has its own url and set of properties.
b) It's replacing a declarative, easy to read style with imperative assignments